src/content/previews/index.mdx

```diff
@@ -16,6 +16,8 @@ Preview environments allow you to include a live preview of your changes on ever
 
 Okteto's preview environments are powered by Kubernetes and easily integrate with your favorite source control and CI/CD provider. They work using the [Okteto manifest](core/okteto-manifest.mdx), so are very simple to get started with.
 
+To unlock the full potential of Okteto Preview Environments, you can also combine it with our [testing infrastructure](reference/okteto-cli.mdx#test) and run integration test against your environments.
```

Comment:
I think we need a place to explain the benefits of Okteto Test, @codyjlandstrom where do you think it would be a good place for it?

Reply:
I think it probably needs it's own section within Preview environments to start. `docs/previews/test`
Another option would be to make it as the same menu level as dev and preview environments for awareness of the feature. We should discuss this as part of the GTM effort